server/views/serviceProviderReferrals/actionPlan/addActivities.njk

```diff
@@ -19,7 +19,6 @@
 
   <form method="post" action="{{ presenter.addActivityAction }}">
     <input type="hidden" name="_csrf" value="{{csrfToken}}">
-    <input type="hidden" name="desired-outcome-id" value="{{ presenter.firstDesiredOutcomeId }}">
```

Comment:
Can we get rid of this property on the presenter?

Reply:
Oops forgot - thanks